tpgtools/overrides/compute/samples/networkfirewallpolicyrule/regional.tf.tmpl

```diff
@@ -1,3 +1,13 @@
+resource "google_network_security_address_group" "basic_regional_networksecurity_address_group" {
```

Comment:
@slevenick the files inside the tpgtools/overrides, I'm updating because they will override the examples in the documentation part, that the user can have one example on how to use it. And add them into the tests for the scenarios.